app/__init__.py

```diff
@@ -1,11 +1,14 @@
 from flask import Flask
 from .db import db, migrate
-from .models import task, goal
+from .models import task
+from .models import goal
+from .routes.task_routes import tasks_bp
```

Comment:
Don't forget that the convention for naming blueprints is to name them `bp`. With each blueprint being named the same thing we will need to import them under an alias like so: 

```python
from .routes.task_routes import bp as tasks_bp
```

Though I see you did it for `goal`